sdk/identity/azure-identity/tests/managed-identity-live/service-fabric/sfmitestuser/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+ARG PYTHON_VERSION=2.7
+
+# docker can't tell when the repo has changed and will therefore cache this layer
+FROM alpine/git as repo
+RUN git clone https://github.com/Azure/azure-sdk-for-python --single-branch --branch master --depth 1 /azure-sdk-for-python
+
+
+FROM python:${PYTHON_VERSION}-slim
+
+COPY --from=repo /azure-sdk-for-python/sdk/identity /sdk/identity
+COPY --from=repo /azure-sdk-for-python/sdk/core/azure-core /sdk/core/azure-core
+COPY --from=repo /azure-sdk-for-python/sdk/keyvault/azure-keyvault-secrets /sdk/keyvault/azure-keyvault-secrets
+
+WORKDIR /sdk/identity/azure-identity/tests/managed-identity-live
+RUN pip install --no-cache-dir -r requirements.txt
+
+ENV AZURE_IDENTITY_TEST_VAULT_URL=<your vault URL>  # looks like https://<vault name>.vault.azure.net/
+ENV AZURE_IDENTITY_TEST_MANAGED_IDENTITY_CLIENT_ID=<AdminUser client ID>  # found in the managed identity overview
```

Comment:
Can Service Fabric set these variables at runtime?

Reply:
I'm not sure what you mean by setting them at runtime -- do you mean setting them in a way that doesn't require changing these lines?

Reply:
Yes, is it possible to configure Service Fabric to set environment variables for a container?

Reply:
It does actually look like [this is possible](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-how-to-specify-environment-variables), which I hadn't seen before -- I'll try editing the service manifests and deploying applications with the CLI today